Templates/Files/Local Files/JSON to HubSpot - Companies/long-description.md

```diff
@@ -0,0 +1,45 @@
+This template reads a JSON file containing company data from a local fileshare, and either inserts or updates each company in HubSpot.
+
+![Template](assets/JSON_to_HubSpot_-_Companies.svg)
+
+# Prerequisites
+
+This template assumes that the following prerequisites are in place:
+
+- The Frends agent has access to the local fileshare containing the JSON file.
+- Using the HubSpot API requires an API key.
```

Comment:
As with the other template, I think the better phrasing for "Using the HubSpot API requires an API key" is "HubSpot API key is accessible", since it's more of a prerequisite.